src/refactor_nrepl/util.clj

```diff
@@ -88,3 +88,12 @@
           (maybe-log-exception e)
           ;; return false, because `with-suppressed-errors` is oriented for predicate usage
           false)))))
+
+(defn interrupted?
+  "Has the current thread been interrupted?
+
+  Observing this condition helps `future-cancel` effectively cancel `future`s."
+  ([]
+   (interrupted? ::_))
+  ([_x]
```

Comment:
This looks a bit weird on its own.  I'd call the ignored param simply `_` and maybe even add a comment about having an extra arity for using in predicates.  Otherwise someone is likely to waste 2 minutes trying to see if they can refactor it.